src/controllers/who.is.making.filing.controller.ts

```diff
@@ -24,5 +31,14 @@ export const post = (req: Request, res: Response, next: NextFunction) => {
     nextPageUrl = getUrlWithParamsToPath(DUE_DILIGENCE_WITH_PARAMS_URL, req);
     oeNextPageUrl = getUrlWithParamsToPath(OVERSEAS_ENTITY_DUE_DILIGENCE_WITH_PARAMS_URL, req);
   }
+  console.log('next-page-urls');
+  console.log(req.params);
+  console.log(nextPageUrl);
+  console.log(DUE_DILIGENCE_WITH_PARAMS_URL);
+  console.log(oeNextPageUrl);
+
+  console.log('app-data-new');
+  console.log(getApplicationData(req.session));
+  console.log(req.app);
```

Comment:
Assume these console.log()s will be removed when the PR comes out of draft status.

Reply:
Yes, these will be gotten rid of when we move out of draft status.

Reply:
Actually, I've removed them now. Will be updated on next push.